recipe/install_nvcc.sh

```diff
@@ -42,9 +42,9 @@ if [[ \$(grep -q "CUDA Version ${PKG_VERSION}" \${CUDA_HOME}/version.txt) -ne 0
 fi
 
 export CUDA_HOME="\${CUDA_HOME}"
-export CFLAGS="\${CFLAGS} -I\${CUDA_HOME}/include"
-export CPPFLAGS="\${CPPFLAGS} -I\${CUDA_HOME}/include"
-export CXXFLAGS="\${CXXFLAGS} -I\${CUDA_HOME}/include"
+export CFLAGS="\${CFLAGS} -I\${CUDA_HOME}/include -I/usr/include"
```

Comment:
Mixing `-I/usr/include` and the sysroot is asking for trouble. cc @mingwandroid

Reply:
Admittedly I don't like this either. That said, this is just the headers and we are adding them last after those included in the build and host prefixes.

Reply:
No, you ar`e not. Host prefix is included as `-isystem` which has lower priority than `-I`, therefore you need to make this `-isystem`  instead to make it work. But with `-isystem` if you are adding any folder in a sysroot like folder then you need to add all of the folders including the C++ headers folder, etc because `-isystem` is weird like that. This means you won't be able to use the C++ headers from the new compiler (7.3.0) and will be stuck with 4.8.5

Reply:
Well we could add `-I${PREFIX}/include` and similar here.

Reply:
Even still you are messing with the system includes.

There are 2 options,
1. Symlink all headers cuda needs to a different directory and add it.
2. Use cudatoolkit-dev which has the correct directory structure.

Reply:
> Symlink all headers cuda needs to a different directory and add it.

Reorganizing the Docker image a little bit could make sense. However it does seem like we are going in the opposite direction of CUDA Toolkit development, which is a concern.

> Use cudatoolkit-dev which has the correct directory structure.

Suspect it will have the same issue as the installer itself is placing these files in `/usr`.